compass/ocean/tests/internal_wave/default/__init__.py

```diff
@@ -10,18 +10,26 @@ class Default(TestCase):
     The default test case for the internal wave test
     """
 
-    def __init__(self, test_group):
+    def __init__(self, test_group, vlr=False):
         """
         Create the test case
 
         Parameters
         ----------
         test_group : compass.ocean.tests.internal_wave.InternalWave
             The test group that this test case belongs to
+        vlr : boolean
+            Whether vertical Lagrangian remapping will be tested
         """
-        super().__init__(test_group=test_group, name='default')
+        name = 'default'
+        if vlr:
+            subdir = 'vlr/' + name
```

Comment:
I realize it works but I prefer that we not use string addition.
```suggestion
            subdir = f'vlr/{name}' 
```